mappings/net/minecraft/world/gen/decorator/DecoratorTreeJungleSmall.mapping

```diff
@@ -1,6 +1,6 @@
 CLASS ayi net/minecraft/world/gen/decorator/DecoratorTreeJungleSmall
-	FIELD a logState Lave;
-	FIELD b leafState Lave;
+	FIELD a leavesState2 Lave;
+	FIELD b logState2 Lave;
```

Comment:
Is the `2` suffix necessary? If it's referring to `log2` and `leaves2`, again, aren't they redundant due to the class these fields are in?